### ISSUE TITLE
Taxonomy: Most Used Heading Level

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#9272]
## Bug Description

'Most Used' section of tags should be a level 4 heading not level 3. 

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by csossi** [comment]
Verified in QA
